git-open

```diff
@@ -93,14 +93,18 @@ else
   # custom GitLab
   gitlab_domain=$(git config --get gitopen.gitlab.domain)
   gitlab_port=$(git config --get gitopen.gitlab.port)
+  gitlab_protocol=$(git config --get gitopen.gitlab.protocol)
+  if [ -z "$gitlab_protocol" ]; then
+      gitlab_protocol=http
```

Comment:
the default should be https, no? otherwise the docs in the readme don't make sense as setting it is redudant..